benchmarks/dynamo/torchbench.yaml

```diff
@@ -252,6 +252,7 @@ accuracy:
     eager_not_deterministic:
       # Models that deterministic algorithms can not be turned on for eager mode.
       - Background_Matting
+      - pytorch_unet
```

Comment:
@ezyang @xuzhao9  Let me know what we want to do instead in the benchmark repo:
https://github.com/pytorch/benchmark/pull/2204#discussion_r1547637515